ui/src/app/pages/login/account-creation/account-creation-options.tsx

```diff
@@ -75,6 +76,14 @@ export const AccountCreationOptions = {
     {label: `Not Hispanic or Latino`, value: Ethnicity.NOTHISPANIC},
     {label: ` Prefer not to answer`, value: Ethnicity.PREFERNOANSWER}
   ],
+  gender: [
+    {label: 'Man', value: Gender.MAN},
+    {label: 'Non-Binary', value: Gender.NONBINARY},
+    {label: 'Transgender', value: Gender.TRANSGENDER},
+    {label: 'Woman', value: Gender.WOMAN},
+    {label: 'None of these describe me', value: Gender.NONE},
```

Comment:
I wouldn't name this option `NONE`, because when I read it I assumed it meant "I have no gender (agender)". Which isn't 100% the same as this statement I don't think.